deepchecks/tabular/checks/methodology/__init__.py

```diff
@@ -8,19 +8,23 @@
 # along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
 # ----------------------------------------------------------------------------
 #
-"""Module contains checks for methodological flaws in the model building process."""
-from .boosting_overfit import BoostingOverfit
-from .datasets_size_comparison import DatasetsSizeComparison
-from .date_train_test_leakage_duplicates import DateTrainTestLeakageDuplicates
-from .date_train_test_leakage_overlap import DateTrainTestLeakageOverlap
-from .identifier_leakage import IdentifierLeakage
-from .index_leakage import IndexTrainTestLeakage
-from .model_inference_time import ModelInferenceTime
-from .single_feature_contribution import SingleFeatureContribution
-from .single_feature_contribution_train_test import \
-    SingleFeatureContributionTrainTest
-from .train_test_samples_mix import TrainTestSamplesMix
-from .unused_features import UnusedFeatures
+"""
+Module contains checks for methodological flaws in the model building process.
+
+.. deprecated:: 0.6.5
+        :mod:`deepchecks.tabular.checks.methodology is deprecated and will be removed in deepchecks 0.8 version.
+        Use :mod:`deepchecks.tabular.checks.model_evaluation`, :mod:`deepchecks.tabular.checks.train_test_validation`,
+        :mod:`deepchecks.tabular.checks.integrity` instead.
+
+"""
+import warnings
+
+from ..model_evaluation import BoostingOverfit, ModelInferenceTime, UnusedFeatures
+from ..train_test_validation import (DatasetsSizeComparison, DateTrainTestLeakageDuplicates,
+                                     DateTrainTestLeakageOverlap, IdentifierLeakage, IndexTrainTestLeakage,
+                                     SingleFeatureContributionTrainTest, TrainTestSamplesMix)
+from ..integrity import SingleFeatureContribution
```

Comment:
This wasn't included in the original methodology suite

Reply:
what do you mean? the checks here are the checks that were previously on the methodology folder

Reply:
Oh sorry, I was confused between "in the module" and "in the suite"